bylaws/009-naming-conventions.md

```diff
@@ -0,0 +1,11 @@
+Naming conventions for code released by PHP FIG
+===============================================
+
+1. Interfaces MUST be suffixed by `Interface`: e.g. `Psr\Foo\BarInterface`.
+2. Abstract classes MUST be prefixed by `Abstract`: e.g. `Psr\Foo\AbstractBar`.
+3. Traits MUST be suffixed by `Trait`: e.g. `Psr\Foo\BarTrait`.
+4. PSR-1, 4, and 12 MUST be followed.
+5. For code released as part of a PSR, the vendor namespace MUST be `Psr` and the Composer package name MUST be `psr/<package>` (e.g., `psr/log`).
+6. For code released as part of a PER or any other Auxiliary Resources, the vendor namespace MUST be `Fig` and the Composer package name MUST be `fig/<package>` (e.g., `fig/cache-util`).
```

Comment:
Why not `Per`/`per`?

Reply:
Mainly because `Fig` is already established and in use, and I see little reason to add another namespace for us to manage. :smile: If we were starting over today, I think we'd probably use `fig/` as the namespace for everything, including PSRs.